iocore/net/UnixNetVConnection.cc

```diff
@@ -1514,9 +1514,10 @@ UnixNetVConnection::set_inactivity_timeout(ink_hrtime timeout_in)
 #else
   if (timeout_in == 0) {
```

Comment:
@oknet How about dividing this block to new function like `set_default_inactivity_timeout()`?
Because, in some cases people want to set `inactivity_timeout_in` `0`. ( One of use cases is drain traffic )
And, currently, even if `set_inactivity_timeout()` is called from `cancel_inactivity_timeout()`, the `inactivity_timeout_in` will be `default_inactivity_timeout`. This could accidentally extend inactivity timeout and increase connections, right?